lib/board.rb

```diff
@@ -0,0 +1,63 @@
+class Board
+
+  attr_accessor :cells
+
+  @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+  def initialize
+    reset
+  end
+
+  def reset
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
```

Comment:
This method is redundant. Prefer to use the bang method `reset!` which clearly communicates that internal state is changed by calling  the method.